cmd/psqldef/psqldef_test.go

```diff
@@ -290,6 +290,41 @@ func TestPsqldefAddForeignKey(t *testing.T) {
 	assertApplyOutput(t, createUsers+createPosts+addForeignKey, nothingModified)
 }
 
+func TestPsqldefAddForeignKeyWithAlter(t *testing.T) {
```

Comment:
I'm trying to move any new SQL-level tests to `cmd/psqldef/tests.yml` because that's faster and less redundant. Could you move the tests to that place?